server/storeProfile.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once 'config.php';
+
+storeProfile(_$POST);
+
+function storeProfile($data) {
+  GLOBAL $DB_CONN;
+  $response = array();
+  $response['success'] = false;
+
+  $DB_CONN -> beginTransaction();
+
+  try {
+    $clientInsert = "INSERT INTO vita.user
```

Comment:
You really shouldn't be inserting into the user table. The profile page is strictly for getting and updating profile information. Not for creating users.